testing/unit_testing/count_cohorts_test/test_CountCohorts.pf

```diff
@@ -0,0 +1,98 @@
+module test_CountCohorts
+  !
+  ! DESCRIPTION:
+  !		Tests the patch's CountCohorts method
+  !
+  use FatesConstantsMod, only : r8 => fates_r8
+  use FatesCohortMod,    only : fates_cohort_type
+  use FatesPatchMod,     only : fates_patch_type
+  use funit
+
+  implicit none
+  
+  @TestCase
+  type, extends(TestCase) :: TestCountCohorts
+    type(fates_patch_type), pointer :: patch
+    contains
+      procedure :: setUp
+      procedure :: tearDown
+  end type TestCountCohorts
+
+  real(r8), parameter :: tol = 1.e-13_r8
+
+  contains 
+  
+    subroutine setUp(this)
```

Comment:
It's perhaps premature to comment here since you haven't added the actual unit tests for CountCohorts yet, but I'm anticipating that you'll have multiple tests where you want to test different scenarios, so I'm wondering if the stuff currently in setUp should be moved out of here, so each test can test something different. I like the idea of consolidating this setup stuff into a subroutine rather than having it inline in a test; you could do that by having the test call a named subroutine like "call setup_patch_with_8_cohorts", where you put the current code.

Update: Oh, maybe this is just an artifact of starting with what you had for test_insertCohort.pf, and you plan to rework this.

Reply:
oh okay I like this idea for the other tests as well! I have a lot of tests where a bulk of the test is just creating a hard-coded patch.